WPMedia/fusion-news-theme-blocks
# TMEDIA-443 fix: Ensure accessibility issue is addressed by making inner list item

## Description
Fix accessibility issue from the placeholder list item 

## Jira Ticket
- [TMEDIA-443](https://arcpublishing.atlassian.net/browse/TMEDIA-443)

## Acceptance Criteria
Only list items found in nav chain block's section list 

## Test Steps

1. Checkout this branch `git checkout FIX-accessibility-nav-item-placeholder`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/header-nav-chain-block`
3. Go to http://localhost/homepage/?_website=the-gazette
4. Ensure that placeholder still takes up space and follows fixes in https://github.com/WPMedia/fusion-news-theme-blocks/pull/1010
5. Run lighthouse locally. Ensure that accessibility issue is not found with non-list items in a list 

## Effect Of Changes
### Before

![Screen Shot 2021-08-05 at 12 55 28](https://user-images.githubusercontent.com/5950956/128398059-321d62f4-b5e1-4fa8-a711-5128a09a48c6.png)

![Screen Shot 2021-08-05 at 12 54 24](https://user-images.githubusercontent.com/5950956/128398080-513a14d0-0442-4d41-87b1-7ca5cd9c8ce6.png)


### After

![Screen Shot 2021-08-05 at 09 46 29](https://user-images.githubusercontent.com/5950956/128374369-a1bc3a57-fd61-4b1a-a592-98c80173ea34.png)

![Screen Shot 2021-08-05 at 12 53 33](https://user-images.githubusercontent.com/5950956/128398111-0d69de1d-e7c0-4bb6-bcde-4effe2983703.png)


## Dependencies or Side Effects

Closes https://github.com/WPMedia/fusion-news-theme-blocks/issues/1030

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
